client/keys/add_test.go

```diff
@@ -34,8 +34,10 @@ func Test_runAddCmdBasic(t *testing.T) {
 		kb, err := keys.NewKeyring(sdk.KeyringServiceName(), viper.GetString(flags.FlagKeyringBackend), kbHome, mockIn)
 		require.NoError(t, err)
 		t.Cleanup(func() {
-			kb.Delete("keyname1", "", false)
-			kb.Delete("keyname2", "", false)
+			err := kb.Delete("keyname1", "", false)
+			require.NoError(t, err)
```

Comment:
Hold on, this is a cleanup function - if this fails the other `Delete()` call is not executed. Ignoring `err` here seems legit